tests/test_dfs3.py

```diff
@@ -3,38 +3,30 @@
 from datetime import datetime
 import numpy as np
 from mikeio.dfs3 import Dfs3
+from mikeio import Dfsu
```

Comment:
Do you use the Dfsu class in this file?

Reply:
No I may just wanted to test something. please remove it.